UI/volume-control.cpp

```diff
@@ -1198,9 +1204,8 @@ QRect VolumeMeter::getBarRect() const
 
 void VolumeMeter::changeEvent(QEvent *e)
 {
-	if (e->type() == QEvent::StyleChange) {
+	if (e->type() == QEvent::StyleChange)
 		recalculateLayout = true;
-	}
```

Comment:
Not sure what you mean by "not related". I added changeEvent as part of the PR, to ensure that QSS changes such as font trigger recalc of layout. This update to remove the curlies was per your request, either for this case or elsewhere.

Reply:
The GitHub UI had shown at least part of that block is unchanged, implying it previously existed.  This appears to have been a display bug in the GitHub UI.

Here is what is shows for me in this current thread:
![image](https://user-images.githubusercontent.com/624931/133339278-33bbfe38-c2db-49be-91ca-703762e36adf.png)

Here is what it now shows for me if I view the file in the "Files changed" tab:
![image](https://user-images.githubusercontent.com/624931/133339385-84c54308-8c46-4988-99a7-6ea71b1ba685.png)

My comment was based on the information that the GitHub UI showed to me in the first image.  Since this is clearly a bug in how the GitHub UI displayed changes, please ignore this request.